src/main/java/com/waterloorocketry/airbrakeplugin/simulated/SimulatedDragForceInterpolator.java

```diff
@@ -19,39 +22,54 @@ public static class Data {
 
         /**
          * Constructs a new `Data` with values
+         *
          * @param extension Airbrakes extension value from 0 to 1 inclusive
-         * @param velocity Upwards velocity of the rocket in m/s
-         * @param altitude Altitude of the rocket in m
+         * @param velocity  Upwards velocity of the rocket in m/s
+         * @param altitude  Altitude of the rocket in m
          */
         public Data(double extension, double velocity, double altitude) {
             this.extension = extension;
             this.velocity = velocity;
             this.altitude = altitude;
         }
+
+        @Override
+        public String toString() {
+            return "Data{" +
+                    "extension=" + extension +
+                    ", velocity=" + velocity +
+                    ", altitude=" + altitude +
+                    '}';
+        }
     }
 
-    private final NavigableMap<Double, QuadraticFunction> fs = new TreeMap<>();
-    /**
-     * The height about sea level that simulations were conducted at, in meters
-     */
-    private static final double SIM_ALTITUDE = 1000;
+    private final NavigableMap<Double, Cubic2VariableFunction> points = new TreeMap<>();
 
     /**
      * Constructs a new `SimulatedDragForceInterpolator` with the simulated values
      */
     public SimulatedDragForceInterpolator() {
-        fs.put(0.0, new QuadraticFunction(0.0035, 0.1317, -5.0119));
-        fs.put(0.5, new QuadraticFunction(0.0045, 0.1031, -3.8231));
-        fs.put(1.0, new QuadraticFunction(0.006, 0.1038, -4.2522));
+        points.put(0.0, new Cubic2VariableFunction(232.2951, 244.7010, -75.1435, 64.3402, -79.5220, 11.7309, -0.8306, -20.4344, 9.7041, -0.6148));
+        points.put(0.1, new Cubic2VariableFunction(235.8993, 249.2100, -76.2767, 65.8251, -81.2931, 12.0289, -0.8408, -21.0236, 9.9787, -0.7853));
+        points.put(0.2, new Cubic2VariableFunction(245.6886, 260.2967, -80.2111, 69.3746, -85.4361, 12.5705, -0.6199, -22.1676, 10.4297, -0.6666));
```

Comment:
can we add bit more documentation like the order of vars, units etc, and general explanation? sure u can tell from inspecting other parts of the code but better just make it explicitly ecplained